apps/application-system/form/src/components/AuthenticationLoading/AuthenticationLoading.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react'
+import { Box, LoadingIcon, Stack, Text } from '@island.is/island-ui/core'
+import * as styles from './AuthenticationLoading.treat'
+
+const AuthenticationLoading: FC = () => {
```

Comment:
We'll probably wanna combine these in the UI library in the future

Reply:
yes, I was thinking the same.
Also, the authentication handling is pretty much the same for the applicaion system and the service portal, maybe we should extract some of the code into a lib ? @atligudl , @obmagnusson